src/main/java/seedu/duke/command/EditCommand.java

```diff
@@ -33,8 +39,62 @@ public EditCommand(String input) {
         String[] extractedFields = extract(input);
         this.name = extractedFields[0];
         this.newFolderName = extractedFields[1];
-        this.newDescription = extractedFields[2];
-        this.newName = extractedFields[3];
+        this.newName = extractedFields[2];
+    }
+
+    /**
+     * Asks user for the new fields that will replace the old fields.
+     *
+     * @return The new fields
+     */
+    public String[] inquireFields(Secret secret) {
+        String[] inquiredFields = null;
+
+        if (secret instanceof BasicPassword) {
+            inquiredFields = new String[3];
+            System.out.println("Enter the new Username: ");
```

Comment:
This system should not be like this because it doesnt check alot of things like empty inputs. I will use my inquire functions for this